containers/centos-9.containerfile

```diff
@@ -25,6 +26,9 @@ RUN echo v1 \
         python3.11-setuptools \
         python3.11-ovirt-engine-sdk4 \
         python3.11-pip \
+        python3.12-devel \
+        python3.12-setuptools \
+        python3.12-pip \
```

Comment:
Why not replace python 3.11 with 3.12?

Reply:
Then we can test on python3.12 and 3.11. It doesn't hurt :)